src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/Configuration/OptionListAttributeConfigurationType.php

```diff
@@ -32,13 +32,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('format', 'choice', [
                 'choices' => [
-                    self::FORMAT_DROPDOWN           => 'Dropdown',
-                    self::FORMAT_MULTIPLE_SELECTION => 'Multiple selection',
+                    self::DROPDOWN    => 'Dropdown',
+                    self::MULTISELECT => 'Multiple selection',
                 ],
                 'attr' => [
                     'class' => 'js-format-selector'
                 ],
             ])
+            ->add('expanded', 'checkbox', [
+                'attr' => [
+                    'description' => 'This will dictate if using radio buttons (Dropdown) or multi checkboxes (Multiple selection).'
```

Comment:
To clarify this, we could put:
```
For Single Value, tick to use Radio Buttons instead of a Dropdown
For Multi Value, tick to use Checkboxes instead of a Multi-select
```